test/evp_pkey_provided_test.c

```diff
@@ -98,6 +98,7 @@ static int compare_with_file(const char *alg, int type, BIO *membio)
     if (!TEST_ptr(fullfile))
         goto err;
 
+    fprintf(stderr, "\nReading reference key data from file %s\n", fullfile);
```

Comment:
This should be either removed or use TEST_info() instead.

Reply:
Changed to using `TEST_info()`